pic-domain/src/main/kotlin/com/mashup/pic/domain/user/UserRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.mashup.pic.domain.user
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+
+@Repository
+interface UserRepository : JpaRepository<User, Long> {
+    fun findByOAuthId(oauthId: Long): User?
```

Comment:
```suggestion
    fun findByOAuthIdOrNull(oauthId: Long): User?
```
Kotlin API에서 반환값이 nullable 하면 OrNull 적어주는게 해당 함수의 동작을 파악하기 더 좋은거 같아요

Reply:
이거 함더 봐주세요

Reply:
JPA에서 OrNull 기능을 어떻게 이용할 수 있나용?